core/server/src/test/java/alluxio/master/file/PermissionCheckerTest.java

```diff
@@ -183,11 +185,13 @@ public static void beforeClass() throws Exception {
 
   @AfterClass
   public static void afterClass() throws Exception {
```

Comment:
Is there anything special about this test suite? If not, it would be nice to have a consistent before/after pattern for these security tests. @aaudiber